mlptrain/sampling/md_openmm.py

```diff
@@ -80,31 +116,31 @@ def run_mlp_md_openmm(
 
         (mlt.Trajectory):
     """
-
-    restart = restart_files is not None
-
-    # TODO: Implement this!
-    if copied_substrings is not None:
+    if (
+        fbond_energy is not None
+        or bbond_energy is not None
+        or bias is not None
+    ):
+        # TODO: Implement bias and other types of constraints.
         raise NotImplementedError(
```

Comment:
That's a good point. Thanks for bringing that up. Let me see what I can do.

Additionally, it would definitely be worthwhile to implement metadynamics and/or other enhanced sampling methods in future PRs. Maybe we could discuss at some point which methods would be most useful for you.

Reply:
Absolutely, we can finish this PR without the biasing part as it is already quite large, but I am happy to discuss the next steps.

Reply:
@juraskov, this should now be resolved. A `NotImplementedError` will now be raised before the AL starts if any of the following arguments are present: `constraints`, `fbond_energy`, `bbond_energy`, and `bias`.

Reply:
Thanks, I have tested it and it works fine. I am now happy with the code. I will wait for comments from Hanwen. If there is nothing serious, we can approve it after.